### PR TITLE
Removing serialize_sequence

### PR DESCRIPTION
**Pull request by AntonBikineev**

### CODE REVIEW

File: hpx/runtime/serialization/serialize_sequence.hpp [hunk 1]
```diff
@@ -1,173 +0,0 @@
-//  Copyright (c) 2007-2015 Hartmut Kaiser
```
Comment (sithhell) [inline hpx/runtime/serialization/serialize_sequence.hpp]: I am not sure if we should remove it entirely. Users might still want to be able to serialize fusion sequences.

Comment (hkaiser) [inline hpx/runtime/serialization/serialize_sequence.hpp]: I'd vote for complete removal. We could ask Joel if he would like to accept a patch adding this directly to Boost, though.

Comment (sithhell) [inline hpx/runtime/serialization/serialize_sequence.hpp]: Ok. complete removal it is.

**Comment by sithhell** [comment]
LGTM.

**Pull request merged**